src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -212,5 +212,8 @@ protected Object[][] getContents() {
             {"R_failedFedauth", "Failed to acquire fedauth token: "},
             {"R_noLoginModulesConfiguredForJdbcDriver",
                     "javax.security.auth.login.LoginException (No LoginModules configured for SQLJDBCDriver)"},
-            {"R_unexpectedThreadCount", "Thread count is higher than expected."}};
+            {"R_unexpectedThreadCount", "Thread count is higher than expected."},
+            {"R_classLoaderNotFoundForColumnType", "Class Loader for type {0} not found for column {1}"},
+            {"R_classNotAssignable", "Class {0} is not assignable from class {1} for column {2}"},
+            {"R_expectedClassDoesNotMatchActualClass", "Expected column class {0} does not match actual column class {1} for column {2}."}};
```

Comment:
these are not used

Reply:
Specifically `R_classLoaderNotFoundForColumnType` and `R_classNotAssignable`.

Reply:
Removed